evaluation/swe_bench/EVAL_PATCH.md

```diff
@@ -0,0 +1,256 @@
+# Evaluate Generated Patches
```

Comment:
ditto (this file shouldn't appear in your PR)

Reply:
okay